### PR TITLE
Module 1: Portfolio

### PR DESCRIPTION
**Pull request by yesuf023**
### Personal Portfolio setup mobile version: Portfolio

In this Portfolio, I include the following:-

- Add the toolbar or header to the homepage
- Add the headline section to the homepage content
- Add responsiveness to the Portfolio website

### CODE REVIEW

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @yesuf023 😊,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Invalid Code Review Request 🆘💀
 In the case of README consider not leaving any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement.
Kindly complete your readme section, you can follow the readme in [this repo](https://github.com/AdnanOlarmmi/Technopreneur) for inspiration.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@400;500&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet" />
+    </head>
+<body>
+    <img
+    src="assets/images/Light.svg" alt="status-bar" class="status-bar"/>
+    <header class="header">
```
Comment (DuaneDave) [inline index.html:18]: The top bar seen on the figma was only there to present the design on a mobile phone and it is not necessary for you to design it, so consider removing it to ensure that your UI is as expected. I mean the `img` before the header

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @yesuf023 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Your code is highly semantic
✔️ The UI is appreciable


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (yesuf023) [reply index.html:18]: Great Thanks

Comment (DuaneDave) [reply index.html]: You are welcomed

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  background: none repeat scroll 0 0 transparent;
+  border-spacing: 0;
+  list-style-type: none;
+}
+
+.header {
+  margin: 1%;
+  padding: 1%;
+  width: 100%;
+  display: flex;
```
Comment (DuaneDave) [inline styles.css:19]: [OPTIONAL] Consider removing the margin of 1% and also setting the padding to 1rem, that value makes it look more like the figma design.

**Review by DuaneDave — APPROVED** [review]
Hi @yesuf023 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ The UI looks amazing
✔️ Correct workflow in use

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (yesuf023) [reply index.html]: Awesome, thanks for your feedback that helps me to improve and know technical skills.

**Pull request merged**